src/javascript/jwt/App.jsx

```diff
@@ -26,7 +26,7 @@ function App(props) {
                     </Typography>
                 </Toolbar>
             </AppBar>
-            <ApolloProvider client={client({contextPath: dxContext.context})}>
+            <ApolloProvider client={client({contextPath: window.contextJsParameters.contextPath})}>
                 <DxContext.Provider value={dxContext}>
                     <TokenManager dxContext={dxContext}/>
                 </DxContext.Provider>
```

Comment:
DxContext.Provider is actually not used neither
```suggestion
                <TokenManager dxContext={dxContext}/>
```

Reply:
was not sure, but yes I like to remove unused lines :)

Reply:
you can also remove the DxContxt file :-)